server/game/core/gameSteps/phases/RegroupPhase.ts

```diff
@@ -22,4 +25,23 @@ export class RegroupPhase extends Phase {
             player.drawCardsToHand(2);
         }
     }
+
+    private readyAllCards() {
```

Comment:
Likewise to in the other comment, do we need to do any sort of cardCannot(BeReadied) restrictions here?

Reply:
Isn't needed here since this calls the ReadySystem, and we'll have the check there